x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/transform_messages_pane.tsx

```diff
@@ -47,6 +50,11 @@ export const TransformMessagesPane: React.SFC<Props> = ({ transformId, lastUpdat
   );
 
   const columns = [
+    {
+      name: '',
+      render: (message: TransformMessage) => <JobIcon message={message} />,
+      width: '5%',
```

Comment:
This might be better off with a hardcoded width as on larger screens this ends up with too much space IMO:

![image](https://user-images.githubusercontent.com/7405507/60342444-5c629c80-99a9-11e9-8119-b022b19abaa2.png)

Reply:
Fixed in 35ad48d475c32fd3c7800afabfa2c4bb5985ed99